osm2gtfs/core/stops.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+
+import sys
+import attr
+
+
+@attr.s
+class Station(object):
+
+    osm_id = attr.ib()
+    tags = attr.ib()
+    lat = attr.ib()
+    lon = attr.ib()
+    name = attr.ib()
+
+    stop_id = attr.ib(default="")
+    osm_type = attr.ib(default="relation")
+    location_type = attr.ib(default=1)
+    osm_url = attr.ib(default="http://osm.org/" +
+                      str(osm_type) + "/" + str(osm_id))
+
+    # Stops forming part of this Station
+    _members = attr.ib(default=attr.Factory(list))
+
+    def __attrs_post_init__(self):
+        '''Populates the object with information obtained from the tags
+
+        '''
+        self.osm_url = "http://osm.org/" + str(
+            self.osm_type) + "/" + str(self.osm_id)
+
+    def set_members(self, members):
+        self._members = members
+
+    def get_members(self):
+        return self._members
+
+    def get_stop_id(self):
+        return self.stop_id
+
+    def set_stop_id(self, stop_id):
+        self.stop_id = stop_id
+
+
+@attr.s
+class Stop(object):
+
+    osm_id = attr.ib()
+    osm_type = attr.ib()
+    tags = attr.ib()
+    lat = attr.ib()
+    lon = attr.ib()
+    name = attr.ib()
+
+    stop_id = attr.ib("")
+    location_type = attr.ib(default=0)
+    osm_url = attr.ib(default=None)
+
+    # The id of the Station this Stop might be part of.
+    _parent_station = attr.ib(default=None)
+
+    def __attrs_post_init__(self):
+        '''Populates the object with information obtained from the tags
+
+        '''
+        self.osm_url = "http://osm.org/" + str(
+            self.osm_type) + "/" + str(self.osm_id)
+
+    def set_parent_station(self, identifier, override=False):
+        """
+        Set the parent_station_id on the first time;
+        Second attempts throw a warning
+        """
+        if self._parent_station is None or override is True:
+            self._parent_station = identifier
+        else:
+            sys.stderr.write("Warning: Stop is part of two stop areas:\n")
+            sys.stderr.write(
+                "http://osm.org/" + self.osm_type + "/" + str(
+                    self.osm_id) + "\n")
+            sys.stderr.write("http://osm.org/" + identifier + "\n")
+            sys.stderr.write("http://osm.org/" + self._parent_station + "\n")
```

Comment:
Use `osm_url` here and above.

Reply:
This is not possible, because we are not getting full objects, but rather the parent station is the GTFS `stop_id`, which people likely want to override in their `stops_creator`. Given this I think the error messages leading to OSM objects, are not a good idea here. Taking them out.